hack/unwanted-dependencies.json

```diff
@@ -217,6 +223,8 @@
       ],
       "google.golang.org/appengine": [
         "cloud.google.com/go/compute",
+        "github.com/google/s2a-go",
+        "github.com/grpc-ecosystem/grpc-gateway/v2",
```

Comment:
FYI: this update isn't actually adding a reference from this dep, it just updated its go.mod file past go1.17 and included the indirect reference it already had via oauth